### PR TITLE
Added a shell file to make installation easier

### PR DESCRIPTION
**Pull request by Aneesh02**
Added:
- Procfile
- Aptfile
- Shell file (Makes usage easier)

### CODE REVIEW

File: shell.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+main(){
+	python3 backend/manage.py makemigrations
+	python3 backend/manage.py migrate
+	python3 backend/manage.py runserver
+}
+
+main
```
Comment (KShivendu) [inline shell.sh:9]: This is for users, right? 
Can we call it `setup.sh`? That would be more intuitive. Also, mention this in the README.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.env/*
+*.pyc
+backend/core/__pycache__/
+backend/db.sqlite3
+backend/home/__pycache__/
+backend/home/migrations/__pycache__/*
```
Comment (KShivendu) [inline .gitignore:6]: Just add `__pycache__` and `*.sqlite3`. They shouldn't be included in general.

Comment (Aneesh02) [reply .gitignore:6]: Understood. 👍

File: shell.sh (hunk 1 above)
Comment (Aneesh02) [reply shell.sh:9]: Agreed.

**Pull request merged**